### PR TITLE
fix katib metrics collector bug

### PR DESCRIPTION
**Pull request by YujiOshima**
The manifest of metrics collector needs namespace.

Signed-off-by: YujiOshima <yuji.oshima0x3fd@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1680)

### CODE REVIEW

File: kubeflow/katib/studyjobcontroller.libsonnet [hunk 1]
```diff
@@ -97,7 +97,7 @@
...
+          "defaultMetricsCollectorTemplate.yaml": 'apiVersion: batch/v1beta1\nkind: CronJob\nmetadata:\n  name: {{.WorkerId}}\n  namespace: {{.NameSpace}}  spec:\n  schedule: "*/1 * * * *"\n  successfulJobsHistoryLimit: 1\n  failedJobsHistoryLimit: 1\n  jobTemplate:\n    spec:\n      template:\n        spec:\n          serviceAccountName: metrics-collector\n          containers:\n          - name: {{.WorkerId}}\n            image: katib/metrics-collector\n            args:\n            - "./metricscollector"\n            - "-s"\n            - "{{.StudyId}}"\n            - "-t"\n            - "{{.TrialId}}"\n            - "-w"\n            - "{{.WorkerId}}"\n            - "-n"\n            - "{{.NameSpace}}"\n          restartPolicy: Never',
```
Comment (jlewi) [inline kubeflow/katib/studyjobcontroller.libsonnet]: Can you use a verbatim string
https://jsonnet.org/learning/tutorial.html
using |||
This should allow it to be formated as a multiline string which is more readable.

File: kubeflow/katib/studyjobcontroller.libsonnet [hunk 2]
```diff
@@ -261,7 +290,20 @@
...
+            metadata:
+              name: {{.WorkerId}}
+            spec:
+              template:
+                spec:
+                  containers:
+                  - name: {{.WorkerId}}
+                    image: alpine
+                  restartPolicy: Never
+          |||% {ns: namespace},
```
Comment (jlewi) [inline kubeflow/katib/studyjobcontroller.libsonnet]: Is this a typo? Why is "% {ns:namespace} outside the verbatim string?

**Comment by jlewi** [comment]
Looks like your jsonnet isn't formatted.
/mnt/test-data-volume/kubeflow-presubmit-unittests-1680-35c2780-3655-1551/src/kubeflow/kubeflow/kubeflow/katib/studyjobcontroller.libsonnet is not formatted

You can run
https://github.com/kubeflow/kubeflow/blob/master/scripts/autoformat_jsonnet.sh

**Comment by YujiOshima** [comment]
@jlewi Thanks! I updated!

File: kubeflow/katib/studyjobcontroller.libsonnet (hunk 2 above)
Comment (YujiOshima) [reply kubeflow/katib/studyjobcontroller.libsonnet]: This is not typo. It is string formatting same as example 3 in https://jsonnet.org/ .

**Pull request merged**